js/ui/html_editor/ui.html_editor.js

```diff
@@ -92,7 +92,7 @@ const HtmlEditor = Editor.inherit({
 
             /**
             * @name dxHtmlEditorOptions.mentions
-            * @type dxHtmlEditorMentions
+            * @type Array<dxHtmlEditorMentions>
```

Comment:
```suggestion
            * @type Array<dxHtmlEditorMention>
```
Array of `mention`?
We'll need fix all entries